test/integration/pipeline/CMakeLists.txt

```diff
@@ -3,6 +3,10 @@ target_link_libraries(tx_pipeline_integration_test
     application
     raw_block_insertion
     model_generators
+    cryptography
```

Comment:
redundant, already in _shared_model_ed25519_sha3_

Reply:
ok.